frontend/lib/graphql/graphql_api.graphql.dart

```diff
@@ -32,8 +32,7 @@ class GetRegionsById$Query$Region extends JsonSerializable with EquatableMixin {
 class GetRegionsById$Query extends JsonSerializable with EquatableMixin {
   GetRegionsById$Query();
 
-  factory GetRegionsById$Query.fromJson(Map<String, dynamic> json) =>
-      _$GetRegionsById$QueryFromJson(json);
+  factory GetRegionsById$Query.fromJson(Map<String, dynamic> json) => _$GetRegionsById$QueryFromJson(json);
```

Comment:
These files should be excluded from analyzer and linter.

Reply:
They are, but not from format (they weren't before as well). These changes are just because of changing max line length.

Reply:
They still should be excluded. We could create an extra task for this, but this is the first task which introduces `dart fmt` properly. Changes to these files should only be done by the graphql/protobuf tools.

Reply:
Created an issue: https://github.com/digitalfabrik/ehrenamtskarte/issues/437